src/util.js

```diff
@@ -31,10 +31,11 @@ export function chunkConditional(array, size, predicate) {
 }
 
 export function buildVariableSizeFieldSection(fields) {
+  const sortedFields = [...fields].sort();
```

Comment:
callback for sort() function needs to added here

Reply:
oops 🤠